commit 683ad1c37b7328dd1f2216a6e1c34869d2f92d3b
Author: Mavriq-Dev <1278955+mavriq-dev@users.noreply.github.com>
Date:   Fri May 27 15:31:53 2022 -0400

    patch

diff --git a/luamodule.cpp b/luamodule.cpp
index b4f9d53..183791f 100644
--- a/luamodule.cpp
+++ b/luamodule.cpp
@@ -219,7 +219,7 @@ extern "C" {
 int luaopen_wx(lua_State *L)
 {
     // only initialize the wxLuaState once, allows require to be called more than once
-    if (!s_wxlState.Ok())
+    if (true)
     {
         int argc = 0;
         wxChar **argv = NULL;
